### PR TITLE
rustup: update page, rustup-{show,update,check,default,toolchain,target}: add page

### PR DESCRIPTION
**Pull request by acuteenvy**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

File: pages/common/rustup.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # rustup
 
-> Rust toolchain installer.
 > Install, manage, and update Rust toolchains.
-> More information: <https://github.com/rust-lang/rustup.rs>.
+> Some subcommands, such as `toolchain` `target`, `update`, etc. have their own usage documentation, accessible via `tldr rustup subcommand`.
```
Comment (kbdharun) [inline pages/common/rustup.md:4]:
```suggestion
> Some subcommands, such as `toolchain` `target`, `update`, etc. have their own usage documentation, accessible via `tldr rustup <subcommand>`.
```

**Review by kbdharun — APPROVED** [review]

File: pages/common/rustup-default.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# rustup default
+
+> Set the default Rust toolchain.
+> More information: <https://rust-lang.github.io/rustup>.
+
+- Switch the default Rust toolchain (see `rustup help toolchain` for more information):
+
+`rustup default {{toolchain}}`
```
Comment (mfrw) [inline pages/common/rustup-default.md:8]: In my opinion, can we add a concrete example here ?
e.g `nightly|beta|stable` ?

```suggestion
`rustup default {{nightly|beta|stable}}`
```

**Review by mfrw — APPROVED** [review]
LGTM :)
A teeny nit w.r.t to `rustup default`.

Thank you @acuteenvy for your contribution.

File: pages/common/rustup.md (hunk 1 above)
Comment (acuteenvy) [reply pages/common/rustup.md:4]: That's how it's written in the `git` and `cs` pages.

https://github.com/tldr-pages/tldr/blob/main/pages/common/git.md?plain=1#L4
```markdown
> Some subcommands such as `commit`, `add`, `branch`, `checkout`, `push`, etc. have their own usage documentation, accessible via `tldr git subcommand`.
```

https://github.com/tldr-pages/tldr/blob/main/pages/common/cs.md?plain=1#L4
```markdown
> Some subcommands such as `install`, `launch`, `java`, `fetch`, `resolve`, `complete-dep`, etc. have their own usage documentation, accessible via `tldr cs subcommand`.
```

Should that be changed too?
Or maybe removed altogether, there's many more pages with this kind of notice but without the second part:

```suggestion
> Some subcommands, such as `toolchain`, `target`, `update`, etc. have their own usage documentation.
```

File: pages/common/rustup-default.md (hunk 2 above)
Comment (acuteenvy) [reply pages/common/rustup-default.md:8]: We can, but there's many more things that can be put there.

`rustup toolchain help`
```
standard release channel toolchain names have the following form:

        <channel>[-<date>][-<host>]

        <channel>       = stable|beta|nightly|<major.minor>|<major.minor.patch>
        <date>          = YYYY-MM-DD
        <host>          = <target-triple>

'channel' is a named release channel, a major and minor version
number such as `1.42`, or a fully specified version number, such
as `1.42.0`. Channel names can be optionally appended with an
archive date, as in `nightly-2014-12-18`, in which case the
toolchain is downloaded from the archive for that date.
```

Comment (mfrw) [reply pages/common/rustup-default.md:8]: Sure that makes sense.
I will leave it to your `taste & judgment` :)
Thanks :)

File: pages/common/rustup.md (hunk 1 above)
Comment (kbdharun) [reply pages/common/rustup.md:4]: Yeah, we can remove it altogether, the latter LGTM.

**Pull request merged**